### PR TITLE
feat: Fetch form details from query parameters and auto-generate web app

### PR DESCRIPTION
**Pull request by djcruz93**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
- This feature is integrated to meet the event dashboard feature to automatically generate web app upon clicking generate web app button in event dashboard
- Use the query parameters, fill the form and submit

#### Changes proposed in this pull request:
- Add js script to fetch email and api endpoint and fill the form to generate web app and open the preview in new tab 

Fixes https://github.com/fossasia/open-event-frontend/issues/5476

### CODE REVIEW

**Comment by djcruz93** [comment]
@iamareebjamal @mariobehling can you please review this PR?

File: src/www/js/autoGenerate.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const email = urlParams.get('email');
+    const apiendpoint = urlParams.get('apiendpoint');
```
Comment (iamareebjamal) [inline src/www/js/autoGenerate.js:4]:
```suggestion
    const apiendpoint = urlParams.get('api');
```

File: src/www/js/autoGenerate.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    if(email && apiendpoint){
```
Comment (iamareebjamal) [inline src/www/js/autoGenerate.js:5]:
```suggestion
    if (email && apiendpoint) {
```

File: src/www/js/form.js [hunk 3]
```diff
@@ -472,6 +472,7 @@ function displayButtons(appPath, url) {
       window.open(url);
     }
   });
+  btnLive.click();
```
Comment (iamareebjamal) [inline src/www/js/form.js:475]: Why?

Comment (djcruz93) [reply src/www/js/form.js:475]: When we'll click generate webapp, as suggested by mario in the meeting, the preview must also open in a separate new tab automatically. Let me know if we do not want. @iamareebjamal

Comment (iamareebjamal) [reply src/www/js/form.js:475]: Is this mentioned in the issue?

Comment (djcruz93) [reply src/www/js/form.js:475]: no, opening the preview is not mentioned in the issue.

File: src/www/js/autoGenerate.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
```
Comment (iamareebjamal) [inline src/www/js/autoGenerate.js:1]: This is a very small script, no need to load this over network. Please inline this in the HTML itself

File: src/www/js/form.js (hunk 3 above)
Comment (iamareebjamal) [reply src/www/js/form.js:475]: Then please remove it

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Great!

**Comment by mariobehling** [comment]
It is not working on live site unfortunately. Therefore re-opening issue.

Screenshot from 2020-11-11 08-08-00

**Comment by iamareebjamal** [comment]
That is deployed from master, https://open-event-wsgen-dev.herokuapp.com/ is development branch version

**Pull request merged**